bucket/x64dbg.json

```diff
@@ -43,10 +43,10 @@
         "release\\x64\\plugins"
     ],
     "checkver": {
-        "github": "https://github.com/x64dbg/x64dbg",
+        "url": "https://sourceforge.net/projects/x64dbg/rss?path=/",
```

Comment:
Why not set path=/snapshot?

Precedent

https://github.com/ScoopInstaller/Extras/blob/master/bucket/capture2text.json#L33

Reply:
x64bdg doesn't store anything other than snapshots. I also based my changes on https://github.com/ScoopInstaller/Extras/blob/master/bucket/keepass-plugin-webautotype.json#L16

I amended my changes to include only the snapshots folder.